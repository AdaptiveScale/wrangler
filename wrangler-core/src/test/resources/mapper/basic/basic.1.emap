// This is the simplest test that includes assigning
// fields from input record to output.
out.a = in.a
out.b = in.b;
out.c = in.c;
out.d = in.d;
out.e = in.e;
out.f = in.f;
out.x = in.f; // This was a cross-assignment.