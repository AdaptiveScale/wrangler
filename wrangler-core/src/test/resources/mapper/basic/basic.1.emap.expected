{"a":"test","b":null,"c":1,"d":12,"e":[48,57],"f":4.56,"x":4.56,"rec2a":null,"rec3":[]}